hack/make-rules/test-cmd.sh

```diff
@@ -59,6 +59,7 @@ function run_kube_apiserver() {
     --service-cluster-ip-range="10.0.0.0/24" \
     --token-auth-file=hack/testdata/auth-tokens.csv 1>&2 &
   APISERVER_PID=$!
+  export APISERVER_PID
```

Comment:
OK, so this is really difficult for me to review, and as a maintainer it looks like it introduces more complexity.  Help me out by answering the following: Why are you exporting this? What uses it? What are you fixing by doing this?

As a maintainer, if this export is necessary, this function should have a comment explaining at least what it exports

Reply:
I did not express clearly is my problem, let me answer:
All the variables here and below are unused. using the shellcheck check looks like this:
```
SC2034: SUPPORTED_RESOURCES appears unused. Verify use (or export if used externally).
For more information: https://www.shellcheck.net/wiki/SC2034
```
Because some files might need them, I chose to export these variables. 
https://cs.k8s.io/?q=SUPPORTED_RESOURCES&i=nope&files=&repos=kubernetes/kubernetes
Please feel free to ask me if you have any questions.